index.js

```diff
@@ -87,15 +94,7 @@ function coffeeify(filename, options) {
         header: false
     };
 
-    for (var i = 0, keys = Object.keys(compileOptions); i < keys.length; i++) {
```

Comment:
This should make more sense when you consider that browserify accepts transform options from the command line. See the example in the readme